src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +24,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        LOGGER.info("Successful fetched data from DB. With params userId: {}", userId);
+        LOGGER.debug("Successful fetched data from DB. With params userId: {}", userId);
```

Comment:
info level is used to track business logic. here it is important for business logic that products were fetched from DB